data/web-workers.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * @title Web Workers
+ * @difficulty intermediate
+ * @tags cli, web
+ * @run <url>
```

Comment:
Let's remove this because `deno run ...` command for this example doesn't seem working (ref. https://github.com/denoland/denobyexample/issues/49)

Reply:
SGTM let me do this.